packages/ui-library/src/foundation/semantic-tokens/form.css.ts

```diff
@@ -225,15 +228,15 @@ export const { tokenizedLight: formLight, tokenizedDark: formDark } = renderThem
       flex-direction: column;
 
       &.sm {
-        margin: ${SM.CaptionSlot.Margin};
+        margin: ${forms.captionslot.margin.sm};
       }
 
       &.md {
-        margin: ${MD.CaptionSlot.Margin};
+        margin: ${forms.captionslot.margin.md};
       }
 
       &.lg {
-        margin: ${MD.CaptionSlot.Margin};
+        margin: ${forms.captionslot.margin.md};
```

Comment:
I was about to point out that this could be a mistake, but I actually don't think we use `captionWrapper` any more

Reply:
fixed it anyhow